.travis.yml

```diff
@@ -23,6 +23,9 @@ install:
   - if [ "$CXX" == "g++"  ]; then export CXX="g++-4.9"; fi
   - if [ "$CXX" == "clang++"  ]; then export CXX="clang++-3.5"; fi
 script:
+  - mkdir -p build && cd build/ && cmake .. && make && make install
+  - cd ../bin && echo "os.exit()" | lua -l color_coded && cd ..
+  - rm -rf bin && mv build/clang* . && rm -rf build
   - cmake . && make && make install
-  - echo "os.exit()" | lua -l color_coded
+  - cd bin && echo "os.exit()" | lua -l color_coded
   - make color_coded_test || true
```

Comment:
At this point, we're still in `bin` and we should be at the root leve. Line 30 should be

```bash
cd bin && echo "os.exit()" | lua -l color_coded && cd ..
```